gloo/docs/installation/gateway/docker-compose-consul/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Run Gloo Gateway Locally with Hashicorp Consul & Vault
+menuTitle: Run Gloo with Consul & Vault
+weight: 5
+description: How to run Gloo Locally using Docker-Compose
+---
+
+## Motivation
+
+While Kubernetes provides APIs for config storage ([CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)), credential storage ([Secrets](https://kubernetes.io/docs/concepts/configuration/secret/)), and service discovery ([Services](https://kubernetes.io/docs/concepts/services-networking/service/)), users may wish to run Gloo without using Kubernetes.
+
+Gloo provides alternate mechanisms for configuration, credential storage, and service discovery that do not require Kubernetes, including the use of local `.yaml` files, [Consul Key-Value storage](https://www.consul.io/api/kv.html) and [Vault Key-Value storage](https://www.vaultproject.io/docs/secrets/kv/kv-v2.html).
```

Comment:
Not sure if this is intentional or a copy/paste error. Is Vault also a generic KV store or just for secrets? 
```suggestion
Gloo provides alternate mechanisms for configuration, credential storage, and service discovery that do not require Kubernetes, including the use of local `.yaml` files, [Consul Key-Value storage](https://www.consul.io/api/kv.html) and [Vault Secret storage](https://www.vaultproject.io/docs/secrets/kv/kv-v2.html).
```

Reply:
vault has a number of "secret engines" one of which is a key value store. that's what solo kit uses